server/docs/configuration.md

```diff
@@ -0,0 +1,19 @@
+# Server Configuration
+
+The components of the Hedera Block Node all support loading configuration via the
+environment.
+
+## Default Values
+
+The default configuration allows users to quickly get up and running without having to configure anything. This provides
+ease of use at the trade-off of some insecure default configuration. Most configuration settings have appropriate
+defaults and can be left unchanged. It is recommended to browse the properties below and adjust to your needs.
+
+
+| Environment Variable              | Description                                                                                                   | Default Value |
+|-----------------------------------|---------------------------------------------------------------------------------------------------------------|---------------|
```

Comment:
Just a note, minimizing the content of header lines, instead of aligning them, can make these easier to maintain.

Reply:
✅ I agree about the maintainability for the tables, changed.